README.md

```diff
@@ -1,14 +1,79 @@
 # k8s-mongo-operator
-MongoDB Operator for Kubernetes
+MongoDB Operator for Kubernetes.
 
-## Architecture
-The following diagram shows how the operator communicates with the Kubernetes API and Mongo instances to manage a MongoDB replica set in your cluster.
+[![Docker Build Status](https://img.shields.io/docker/build/ultimaker/k8s-mongo-operator.svg)](https://hub.docker.com/r/ultimaker/k8s-mongo-operator)
 
-![architecture.png](./docs/architecture.png)
+## Features
+The following feature are currently available in this operator:
 
+* Create, update or delete MongoDB replica sets.
+* Automatically initialize the replica set configuration in the master node.
+* Schedule backups to Google Cloud Storage using a Google service account and `mongodump`.
 
-## Running it locally
-- Make sure you have [Docker](https://store.docker.com/search?type=edition&offering=community) running.
-- Make sure you have [MiniKube v0.25.2](https://github.com/kubernetes/minikube/releases/tag/v0.25.2) running.
-- Run `./buld-and-deploy-local.sh`
-  - This will build, deploy the operator and stream the pod logs into the output.
+## Cluster interaction
+Please refer to our [simplified diagram](./docs/architecture.png) to get an overview of the operator interactions with your Kubernetes cluster.
+
+## Deployment
+To deploy this operator in your own cluster, you'll need to create some configuration files.
+An example of these configuration files can be found in [kubernetes/operators/mongo-operator](./kubernetes/operators/mongo-operator)
+
+As you can see there is a service account (mongo-operator-service-account) which has some specific permissions in the cluster.
+These permissions are registered in the cluster role and cluster role binding.
+
+Lastly there is a deployment configuration to deploy the actual operator.
+Usually you'd use an image value like `ultimaker/k8s-mongo-operator:master`, or a specific version.
+All available tags can be found on [Docker Hub](https://hub.docker.com/r/ultimaker/k8s-mongo-operator/).
+
+## Creating a Mongo object
+To deploy a new replica set in your cluster using the operator, create a Kubernetes configuration file similar to this:
+
+```yaml
```

Comment:
:thinking: i'm not sure how to best make sure this example is updated on later versions...